src/public/app/widgets/sync_status.ts

```diff
@@ -4,6 +4,7 @@ import ws from "../services/ws.js";
 import options from "../services/options.js";
 import syncService from "../services/sync.js";
 import { escapeQuotes } from "../services/utils.js";
+import { Tooltip } from "bootstrap";
```

Comment:
This causes the issue described in https://github.com/TriliumNext/Notes/pull/1168#issuecomment-2653398162.

Reply:
agreed, please hold off until I get that fixed
(unfortunatley I cannot seem to put the PR back into Draft anymore... sorry about that :-))